### PR TITLE
Dev ob core api

### LINKED ISSUE #1: read an audio file and use ML model proof of concept
implement basic algorithm to read ML model with Keras, read an audio file, convert the audio data into ML model input layer format, make a prediction and display the prediction

### LINKED ISSUE #3: implement pre-commit hooks
self explanatory

### PR DESCRIPTION
**Pull request by olivier-2018**
implement the following issues:
- proof of concept, make a prediction with a ML model
- pre-commit for text formatting

### CODE REVIEW

File: src_api/__init__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+from src_api import views
+
+# logging.basicConfig(level=logging.DEBUG)
+# logging.basicConfig(filename='app.log', level=logging.DEBUG,format=’%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s’)
+
+app.config["APP_FOLDER"] = os.path.abspath(os.path.join(app.root_path, os.pardir))
+app.logger.info("APP FOLDER: " + app.config["APP_FOLDER"])
+
+app.config["UPLOAD_FOLDER"] = app.config["APP_FOLDER"] + "/static/client/wav"
```
Comment (4982) [inline src_api/__init__.py]: image

an issue with backward and forward slash?

**Review by 4982 — COMMENTED** [review]
Hi Olivier 
i'm not sure but maybe we have an issue with backward and forward slashes (see comment on src_api/__init__.py

File: src_api/__init__.py (hunk 1 above)
Comment (olivier-2018) [reply src_api/__init__.py]: I see... the problem is that the path is hard coded and you run the app from windows powershell where the path is handled in a different way than under linux.  
Have you tried to run in git bash? It should not raise any problem...
In the long run, I think we can replace any hard coded paths and use python path libraries like path.join instead. 
It's only a small issue but I cannot troubleshoot as i am running under linux.
In the interest of time do you want to go ahead with the merge and try to update the path statement in a separate dev branch? 
For info the Heroku production environment is linux so the app should deploy correctly.

Comment (olivier-2018) [reply src_api/__init__.py]: Angelo, I pushed a new version of the dev branch which does not have any hard coded path.. you can pull it and try if it works in powershell..

Comment (4982) [reply src_api/__init__.py]: The path-format is now fine, but the file seems unavailable after choosing it (can't see it after choosing it, we can fix it later)

**Review by 4982 — APPROVED** [review]

**Pull request merged**